package.json

```
@@ -26,7 +26,7 @@
         "ejs": "2.6.1",
         "express": "4.16",
         "lru-cache": "5.1.1",
-        "mdn-browser-compat-data": "0.x",
+        "mdn-browser-compat-data": "^1.0.0",
```

To&nbsp;match the&nbsp;`mdn‑data`&nbsp;query, you&nbsp;should&nbsp;do:

```suggestion
        "mdn-browser-compat-data": "1.x",
```

Alternatively, you&nbsp;could also&nbsp;change the&nbsp;`mdn‑data`&nbsp;query to&nbsp;be&nbsp;`^2.0.6`.

---

Good idea, let's be consistent! I'm going to go with what `npm install` does automatically, which is the `^2.0.6` style semver. Thanks for the review @ExE-Boss.